src/exception_control/win32_exception.cpp

```diff
@@ -108,9 +108,15 @@ std::string win32_exception::format_message(DWORD result)
   std::string message; // std::string version of the formatted message
 
   // Attempt to format the message from the current module resources
+#ifndef TARGET_WINDOWS_STORE
   DWORD cchformatted = FormatMessageA(
       FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
       nullptr, result, GetThreadUILanguage(), reinterpret_cast<LPSTR>(&formatted), 0, nullptr);
```

Comment:
Could the `GetThreadUILanguage()`also be done for desktop windows? Then we could get rid of the `#ifdef`

Reply:
Have set for both to 0 and the #ifndef removed.
Also on the other comment I changed the warning text.